questionnaire-ja.json

```diff
@@ -1,434 +1,434 @@
-{
-  "specVersion": "2.1.1",
-  "language": "ja",
-  "title": "Conformance in Questions",
-  "contextText": "Context",
-  "questionsText": "Questions",
-  "completedText": "Completed?",
-  "statusText": "Status",
-  "answerColumnText": "Answer",
-  "sectionColumnText": "Section",
-  "numberColumnText": "Number",
-  "specRefColumnText": "Spec Ref",
-  "questionColumnText": "Question Text",
-  "tocText": "Table of Contents",
-  "preambleText": "The OpenChain Project builds trust in open source by making license compliance simpler and more consistent. It maintains the OpenChain Specification to define a core set of requirements every quality compliance program must satisfy. The result is that open source license compliance becomes more predictable, understandable and efficient for participants of the software supply chain.\nThis document contains a series of questions to determine whether a company has an OpenChain Conformant program. If each of these questions can be answered with a \"yes\" then that company meets all the requirements of conformance to the OpenChain Specification version 2.1 (functionally identical to ISO/IEC DIS 5230:2020(e)). If any of the questions are answered with a \"no\" then the company can clearly identify where additional investment is needed to improve the compliance process.",
-  "sections": [
-    {
-      "name": "Part 1",
-      "title": "1. Program foundation",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented policy governing the open source license compliance of the Supplied Software?",
-          "number": "1.a",
-          "type": "YES_NO",
+{
+  "specVersion": "2.1.1",
+  "language": "ja",
+  "title": "適合チェック質問一覧",
+  "contextText": "背景",
+  "questionsText": "質問",
+  "completedText": "終了しましたか？",
+  "statusText": "ステータス",
+  "answerColumnText": "回答",
+  "sectionColumnText": "セクション",
+  "numberColumnText": "番号",
+  "specRefColumnText": "仕様参考",
+  "questionColumnText": "質問本文",
+  "tocText": "目次",
+  "preambleText": "OpenChainプロジェクトは、オープンソースライセンスへのコンプライアンスをよりシンプルにし、一貫性を持たせることで、オープンソースに対する信頼を築きます。OpenChain仕様では、各品質コンプライアンスプログラムが満足すべき核となる要件を定義しています。その結果、オープンソース・ライセンス・コンプライアンスがソフトウェアサプライチェーンの中にいる者にとって、より予測可能でわかりやすく、効率的なものになっています。\nこのドキュメントには企業がOpenChain適合がどうかを判断するための一連の質問が記載されています。すべての質問が「はい」だった場合、その企業はOpenChain仕様2.0版のすべての要件に適合していることになります。いくつかの質問に対する答えが「いいえ」だった場合、その企業はコンプライアンスプロセスの改善のために何が必要かを明確にすることができます。",

```

Comment:
Is this missing ISO \/ IEC DIS 5230: 2020 (e)?

Reply:
It looks like the files for JA and KO got mixed up during translation. Fixing now.